src/dune_rules/modules_field_evaluator.ml

```diff
@@ -19,23 +19,30 @@ type kind =
 
 let eval =
   let key = function
-    | Error s -> s
-    | Ok m -> Module.Source.name m
+    | Error s -> [ s ]
+    | Ok m -> [ Module.Source.name m ]
   in
-  let module Unordered = Ordered_set_lang.Unordered (Module_name) in
+  let module Key = struct
```

Comment:
why are this module and `Unordered` defined inline?

Reply:
They aren't used outside of the `eval` function.